docs/components/plugins/APISection.tsx

```diff
@@ -0,0 +1,198 @@
+  methodEntry.signatures.map(signature => {
+    const { name, parameters, comment, type } = signature;
+    return (
+      <>
+        <H3>
+          <InlineCode>{name}()</InlineCode>
+        </H3>
+        {parameters ? <H4>Arguments</H4> : null}
+        {parameters ? (
+          <UL>
+            {parameters?.map(p => (
+              <LI key={`param-${p.name}`}>
+                <InlineCode>
+                  {p.name}: {resolveTypeName(p.type)}
+                </InlineCode>
+              </LI>
+            ))}
+          </UL>
+        ) : null}
+        {comment?.shortText ? (
+          <ReactMarkdown renderers={renderers}>{comment.shortText}</ReactMarkdown>
+        ) : null}
+        <br />
+        {comment?.returns ? <H4>Returns</H4> : null}
+        {comment?.returns ? (
+          <UL>
+            <LI returnType>
+              <InlineCode>{resolveTypeName(type)}</InlineCode>
+            </LI>
+          </UL>
+        ) : null}
+        {comment?.returns ? (
+          <ReactMarkdown renderers={renderers}>{comment.returns}</ReactMarkdown>
+        ) : null}
+        <hr />
+      </>
+    );
+  });
+
+const renderEnum = (enumEntry: object): JSX.Element => {
+  const { name, children, comment } = enumEntry;
+  return (
+    <>
+      <H3>
+        <InlineCode>{name}</InlineCode>
+      </H3>
+      <UL>
+        {children.map(enumValue => (
+          <LI key={enumValue.name}>
+            <InlineCode>
+              {name}.{enumValue.name}
+            </InlineCode>
+            {comment ? ` - ${comment.shortText}` : null}
+          </LI>
+        ))}
+      </UL>
+    </>
+  );
+};
+
+const renderType = (typeEntry: object): JSX.Element => {
+  const { name, comment, type } = typeEntry;
+  return (
+    <>
+      <H3>
+        <InlineCode>{name}</InlineCode>
+      </H3>
+      {comment ? (
+        <P>
+          <ReactMarkdown renderers={renderers}>{comment.shortText}</ReactMarkdown>
+        </P>
+      ) : null}
+      <table>
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th>Description</th>
+          </tr>
+        </thead>
+        <tbody>
+          {type.declaration.children.map(typeProperty => (
+            <tr key={typeProperty.name}>
+              <td>
+                <B>{typeProperty.name}</B>
+                {typeProperty.flags?.isOptional ? (
+                  <>
+                    <br />
+                    <span css={STYLES_OPTIONAL}>(optional)</span>
+                  </>
+                ) : null}
+              </td>
+              <td>
+                <InlineCode>{resolveTypeName(typeProperty.type)}</InlineCode>
+              </td>
+              <td>{typeProperty.comment ? typeProperty.comment.shortText : '-'}</td>
+            </tr>
+          ))}
+        </tbody>
+      </table>
+    </>
+  );
+};
+
+const processData = (packageName: string, version: string = 'unversioned'): JSX.Element => {
```

Comment:
```suggestion
const renderAPI = (packageName: string, version: string = 'unversioned'): JSX.Element => {
```
I would rather go with naming function after what's happening in it 😉